crates/zed/src/languages/ruby/highlights.scm

```diff
@@ -70,10 +70,23 @@
   "**" @operator
 ) @constant.builtin
 
+(constant) @type
```

Comment:
I moved this above ((constant) @constant as I believe it gets overridden if not?